utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -88,6 +88,22 @@ class UrlImageParser private constructor(
             drawableHeight = (drawableHeight.toDouble() * multipleFactor).toInt()
             drawableWidth = (drawableWidth.toDouble() * multipleFactor).toInt()
           }
+          val maximumImageSize = context.resources.getDimensionPixelSize(R.dimen.maximum_image_size)
+          if (drawableHeight >= maximumImageSize || drawableWidth >= maximumImageSize) {
+            // The multipleFactor value is used to make sure that the aspect ratio of the image remains the same.
+            // Example: Height is 420, width is 440 and maximumImageSize is 200.
+            // Then multipleFactor will be (200/440).
+            // The new height will be 191 and new width will be 200.
+            val multipleFactor = if (drawableHeight >= drawableWidth) {
+              // If height is less then the multipleFactor, value is determined by height.
```

Comment:
Update this to: 
`// If height is greater then the width, multipleFactor value is determined by height.`

Similarly in line 82 also this was incorrect,
`// If height is less then the width, multipleFactor value is determined by height.`

Ditto for else statement in both cases.

Reply:
Done